cmd/create/oidcprovider/cmd.go

```diff
@@ -265,3 +278,30 @@ func sha1Hash(data []byte) string {
 	hashed := hasher.Sum(nil)
 	return hex.EncodeToString(hashed)
 }
+
+func validateOperatorRoles(awsClient aws.Client, cluster *cmv1.Cluster) ([]string, error) {
+	var missingRoles []string
+
+	operatorIAMRoles := cluster.AWS().STS().OperatorIAMRoles()
+
+	if len(operatorIAMRoles) == 0 {
+		return missingRoles, fmt.Errorf("No Operator IAM roles found for cluster %s", cluster.Name())
```

Comment:
/nitpick
surround the %s with single quotes: `No Operator IAM roles found for cluster '%s'`